anchor/tests/concert-x.ts

```diff
@@ -76,6 +76,7 @@ describe("concert-x", () => {
 
       const updatedConcertAccount = await program.account.concert.fetch(concertXPda);
       expect(updatedConcertAccount.currentAmount.toNumber()).to.equal(contributionAmount);
+      expect(updatedConcertAccount.contributors.length).greaterThan(0);
```

Comment:
Maybe we could check that contributors Pubkey is in the vec instead? Something like copilot suggests above:
ctx.accounts.concert.contributors.contains(&ctx.accounts.backer.key())